source/extensions/filters/network/dubbo_proxy/serialization.h

```diff
@@ -0,0 +1,94 @@
+#pragma once
+
+#include "envoy/buffer/buffer.h"
+
+#include "common/singleton/const_singleton.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace DubboProxy {
+
+/**
+ * Names of available serialization implementations.
+ */
+class SerializerNameValues {
+public:
+  // hessian serialization
+  const std::string HESSIAN = "hessian";
```

Comment:
nit: prefer CamelCase for short constant

Reply:
fixed